### ISSUE TITLE
Testflight and production crash on iOS devices

### ISSUE THREAD

**Issue by FacuBenito** [software-mansion/react-native-reanimated#6505]
### Description

I encountered a crash while using React Native Reanimated with the Hermes JavaScript engine. The crash consistently occurs when animations or gesture handlers (managed by Reanimated) are executed, and is related to a JSI (JavaScript Interface) exception. The exception is thrown on the native side, as seen in the following error: `C++ Exception: N8facebook3jsi7JSErrorE`. I can't pinpoint where this is happening. However, I can tell you that the feature includes GestureHandler and reanimated library version 3.10. The snack I'll show you is the exact implementation of the relevant parts of the component.

### Steps to reproduce

I have a sort of "Wheel" that can be spun either by clicking on an element inside it or by panning inside it with my finger. When I do that it crashes. Sometimes after a few seconds or even minutes and sometimes right away. Here's the sentry issue that was created for this: https://lila-studio.sentry.io/share/issue/13b5458ac07d4f5ead9ed78857fd36e7/

### Snack or a link to a repository

https://snack.expo.dev/@vixon/training-wheel

### Reanimated version

3.10.1

### React Native version

0.74.5

### Platforms

iOS

### JavaScript runtime

None

### Workflow

Expo Dev Client

### Architecture

Fabric (New Architecture)

### Build type

Other (please specify)

### Device

Real device

### Device model

iPad pro 11-inch

### Acknowledgements

Yes

**Comment by SebaFuchi** [comment]
i have the same problem, there is some news?

**Comment by AgusG77** [comment]
I have the same issue!

**Comment by vamorilla** [comment]
Me too, any news for this?

**Comment by PatricioHeduan** [comment]
Omg its hapening to me too, Needing help here!

**Comment by FacuBenito** [comment]
After fiddling around with some issues on this repo I found some patches for 3.7.1 that I thought I'd give a try. I implemented [this patch](https://github.com/software-mansion/react-native-reanimated/issues/5673#issuecomment-1966955946). And now there's another error which I don't know if it means I'm moving forwards or backwards. The error is `EXC_BAD_ACCESS
`. I'll leave the [Sentry issue](https://lila-studio.sentry.io/share/issue/036d06fef93d4084abf755cf7871447d/) in case it is useful

**Comment by CTOverton** [comment]
Been trying for hours to debug this. 

I keep hitting this error

```
com.facebook.react.JavaScript (13): EXC_BAD_ACCESS (code=1, address=0x20)
```



<details>
  <summary>Full stack trace</summary>
  
  ```
#0	0x0000000103491f48 in std::__1::__function::__value_func<facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)>::__value_func[abi:ue170006](std::__1::__function::__value_func<facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)> const&) at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:445
#1	0x00000001034a08b4 in std::__1::__function::__value_func<facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)>::__value_func[abi:ue170006](std::__1::__function::__value_func<facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)> const&) [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:444
#2	0x00000001034a08ac in std::__1::function<facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)>::function(std::__1::function<facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)> const&) [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:1089
#3	0x00000001034a08ac in std::__1::function<facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)>::function(std::__1::function<facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)> const&) [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:1089
#4	0x00000001034a08ac in reanimated::ShareableHostFunction::ShareableHostFunction(facebook::jsi::Runtime&, facebook::jsi::Function) at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/node_modules/react-native-reanimated/Common/cpp/SharedItems/Shareables.h:217
#5	0x00000001034a07b8 in reanimated::ShareableHostFunction::ShareableHostFunction(facebook::jsi::Runtime&, facebook::jsi::Function) [inlined] at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/node_modules/react-native-reanimated/Common/cpp/SharedItems/Shareables.h:221
#6	0x00000001034a07b0 in std::__1::__shared_ptr_emplace<reanimated::ShareableHostFunction, std::__1::allocator<reanimated::ShareableHostFunction>>::__shared_ptr_emplace[abi:ue170006]<facebook::jsi::Runtime&, facebook::jsi::Function>(std::__1::allocator<reanimated::ShareableHostFunction>, facebook::jsi::Runtime&, facebook::jsi::Function&&) at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__memory/shared_ptr.h:303
#7	0x00000001034a0750 in std::__1::__shared_ptr_emplace<reanimated::ShareableHostFunction, std::__1::allocator<reanimated::ShareableHostFunction>>::__shared_ptr_emplace[abi:ue170006]<facebook::jsi::Runtime&, facebook::jsi::Function>(std::__1::allocator<reanimated::ShareableHostFunction>, facebook::jsi::Runtime&, facebook::jsi::Function&&) [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__memory/shared_ptr.h:292
#8	0x00000001034a0744 in std::__1::shared_ptr<reanimated::ShareableHostFunction> std::__1::allocate_shared[abi:ue170006]<reanimated::ShareableHostFunction, std::__1::allocator<reanimated::ShareableHostFunction>, facebook::jsi::Runtime&, facebook::jsi::Function, void>(std::__1::allocator<reanimated::ShareableHostFunction> const&, facebook::jsi::Runtime&, facebook::jsi::Function&&) at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__memory/shared_ptr.h:1023
#9	0x000000010349d248 in std::__1::shared_ptr<reanimated::ShareableHostFunction> std::__1::make_shared[abi:ue170006]<reanimated::ShareableHostFunction, facebook::jsi::Runtime&, facebook::jsi::Function, void>(facebook::jsi::Runtime&, facebook::jsi::Function&&) [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__memory/shared_ptr.h:1032
#10	0x000000010349d234 in reanimated::makeShareableClone(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const&, facebook::jsi::Value const&) at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/node_modules/react-native-reanimated/Common/cpp/SharedItems/Shareables.cpp:57
#11	0x0000000103491fc8 in std::__1::__function::__value_func<facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)>::operator()[abi:ue170006](facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*&&, unsigned long&&) const [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:518
#12	0x0000000103491fac in std::__1::function<facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)>::operator()(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long) const at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:1169
#13	0x0000000124409480 in ___lldb_unnamed_symbol838 ()
#14	0x000000012441cd70 in ___lldb_unnamed_symbol1219 ()
#15	0x000000012443bda8 in ___lldb_unnamed_symbol1739 ()
#16	0x000000012443b2b4 in ___lldb_unnamed_symbol1738 ()
#17	0x000000012441d018 in ___lldb_unnamed_symbol1225 ()
#18	0x000000012441afc4 in ___lldb_unnamed_symbol1198 ()
#19	0x0000000124455294 in ___lldb_unnamed_symbol1926 ()
#20	0x000000012443e27c in ___lldb_unnamed_symbol1739 ()
#21	0x000000012443b2b4 in ___lldb_unnamed_symbol1738 ()
#22	0x000000012441d018 in ___lldb_unnamed_symbol1225 ()
#23	0x000000012441afc4 in ___lldb_unnamed_symbol1198 ()
#24	0x0000000124454e20 in ___lldb_unnamed_symbol1924 ()
#25	0x000000012443dcd8 in ___lldb_unnamed_symbol1739 ()
#26	0x000000012443b2b4 in ___lldb_unnamed_symbol1738 ()
#27	0x000000012441d018 in ___lldb_unnamed_symbol1225 ()
#28	0x000000012441c7ac in ___lldb_unnamed_symbol1212 ()
#29	0x000000012443bdc4 in ___lldb_unnamed_symbol1739 ()
#30	0x000000012443b2b4 in ___lldb_unnamed_symbol1738 ()
#31	0x000000012441d018 in ___lldb_unnamed_symbol1225 ()
#32	0x000000012441b860 in ___lldb_unnamed_symbol1203 ()
#33	0x00000001244ea8e4 in ___lldb_unnamed_symbol3738 ()
#34	0x000000012441cd70 in ___lldb_unnamed_symbol1219 ()
#35	0x000000012443bda8 in ___lldb_unnamed_symbol1739 ()
#36	0x000000012443b2b4 in ___lldb_unnamed_symbol1738 ()
#37	0x000000012441d018 in ___lldb_unnamed_symbol1225 ()
#38	0x000000012441c7ac in ___lldb_unnamed_symbol1212 ()
#39	0x0000000124401dd4 in ___lldb_unnamed_symbol680 ()
#40	0x00000001039f4bc8 in facebook::jsi::RuntimeDecorator<facebook::jsi::Runtime, facebook::jsi::Runtime>::call(facebook::jsi::Function const&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long) [inlined] at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/ios/Pods/Headers/Public/hermes-engine/jsi/decorator.h:340
#41	0x00000001039f4ba4 in facebook::jsi::WithRuntimeDecorator<facebook::react::(anonymous namespace)::ReentrancyCheck, facebook::jsi::Runtime, facebook::jsi::Runtime>::call(facebook::jsi::Function const&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long) at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/ios/Pods/Headers/Public/hermes-engine/jsi/decorator.h:753
#42	0x00000001038d973c in facebook::jsi::Function::call(facebook::jsi::Runtime&, facebook::jsi::Value const*, unsigned long) const [inlined] at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/ios/Pods/Headers/Public/hermes-engine/jsi/jsi-inl.h:259
#43	0x00000001038d9710 in facebook::jsi::Function::call(facebook::jsi::Runtime&, std::initializer_list<facebook::jsi::Value>) const [inlined] at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/ios/Pods/Headers/Public/hermes-engine/jsi/jsi-inl.h:264
#44	0x00000001038d9710 in facebook::jsi::Value facebook::jsi::Function::call<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, facebook::jsi::Value>(facebook::jsi::Runtime&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, facebook::jsi::Value&&) const at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/ios/Pods/Headers/Public/hermes-engine/jsi/jsi-inl.h:272
#45	0x00000001038d95a8 in facebook::react::JSIExecutor::callFunction(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, folly::dynamic const&)::$_5::operator()() const [inlined] at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/node_modules/react-native/ReactCommon/jsiexecutor/jsireact/JSIExecutor.cpp:253
#46	0x00000001038d9574 in decltype(std::declval<facebook::react::JSIExecutor::callFunction(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, folly::dynamic const&)::$_5&>()()) std::__1::__invoke[abi:ue170006]<facebook::react::JSIExecutor::callFunction(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, folly::dynamic const&)::$_5&>(facebook::react::JSIExecutor::callFunction(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, folly::dynamic const&)::$_5&) [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__type_traits/invoke.h:340
#47	0x00000001038d9574 in void std::__1::__invoke_void_return_wrapper<void, true>::__call[abi:ue170006]<facebook::react::JSIExecutor::callFunction(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, folly::dynamic const&)::$_5&>(facebook::react::JSIExecutor::callFunction(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, folly::dynamic const&)::$_5&) [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__type_traits/invoke.h:415
#48	0x00000001038d9574 in std::__1::__function::__alloc_func<facebook::react::JSIExecutor::callFunction(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, folly::dynamic const&)::$_5, std::__1::allocator<facebook::react::JSIExecutor::callFunction(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, folly::dynamic const&)::$_5>, void ()>::operator()[abi:ue170006]() [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:193
#49	0x00000001038d9574 in std::__1::__function::__func<facebook::react::JSIExecutor::callFunction(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, folly::dynamic const&)::$_5, std::__1::allocator<facebook::react::JSIExecutor::callFunction(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, folly::dynamic const&)::$_5>, void ()>::operator()() at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:364
#50	0x000000010350bf1c in decltype(std::declval<void (*&)(std::__1::function<void ()> const&, std::__1::function<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> ()>)>()(std::declval<std::__1::function<void ()> const&>(), std::declval<std::__1::function<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> ()>>())) std::__1::__invoke[abi:ue170006]<void (*&)(std::__1::function<void ()> const&, std::__1::function<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> ()>), std::__1::function<void ()> const&, std::__1::function<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> ()>>(void (*&)(std::__1::function<void ()> const&, std::__1::function<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> ()>), std::__1::function<void ()> const&, std::__1::function<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> ()>&&) [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__type_traits/invoke.h:340
#51	0x000000010350befc in void std::__1::__invoke_void_return_wrapper<void, true>::__call[abi:ue170006]<void (*&)(std::__1::function<void ()> const&, std::__1::function<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> ()>), std::__1::function<void ()> const&, std::__1::function<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> ()>>(void (*&)(std::__1::function<void ()> const&, std::__1::function<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> ()>), std::__1::function<void ()> const&, std::__1::function<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> ()>&&) at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__type_traits/invoke.h:415
#52	0x00000001038d65c4 in std::__1::__function::__value_func<void (std::__1::function<void ()> const&, std::__1::function<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> ()>)>::operator()[abi:ue170006](std::__1::function<void ()> const&, std::__1::function<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> ()>&&) const [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:518
#53	0x00000001038d65a4 in std::__1::function<void (std::__1::function<void ()> const&, std::__1::function<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> ()>)>::operator()(std::__1::function<void ()> const&, std::__1::function<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> ()>) const [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:1169
#54	0x00000001038d65a4 in facebook::react::JSIExecutor::callFunction(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, folly::dynamic const&) at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/node_modules/react-native/ReactCommon/jsiexecutor/jsireact/JSIExecutor.cpp:251
#55	0x00000001035e607c in std::__1::__function::__value_func<void (facebook::react::JSExecutor*)>::operator()[abi:ue170006](facebook::react::JSExecutor*&&) const [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:518
#56	0x00000001035e6064 in std::__1::function<void (facebook::react::JSExecutor*)>::operator()(facebook::react::JSExecutor*) const [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:1169
#57	0x00000001035e6064 in facebook::react::NativeToJsBridge::runOnExecutorQueue(std::__1::function<void (facebook::react::JSExecutor*)>)::$_8::operator()() const [inlined] at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/node_modules/react-native/ReactCommon/cxxreact/NativeToJsBridge.cpp:310
#58	0x00000001035e604c in decltype(std::declval<facebook::react::NativeToJsBridge::runOnExecutorQueue(std::__1::function<void (facebook::react::JSExecutor*)>)::$_8&>()()) std::__1::__invoke[abi:ue170006]<facebook::react::NativeToJsBridge::runOnExecutorQueue(std::__1::function<void (facebook::react::JSExecutor*)>)::$_8&>(facebook::react::NativeToJsBridge::runOnExecutorQueue(std::__1::function<void (facebook::react::JSExecutor*)>)::$_8&) [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__type_traits/invoke.h:340
#59	0x00000001035e604c in void std::__1::__invoke_void_return_wrapper<void, true>::__call[abi:ue170006]<facebook::react::NativeToJsBridge::runOnExecutorQueue(std::__1::function<void (facebook::react::JSExecutor*)>)::$_8&>(facebook::react::NativeToJsBridge::runOnExecutorQueue(std::__1::function<void (facebook::react::JSExecutor*)>)::$_8&) [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__type_traits/invoke.h:415
#60	0x00000001035e604c in std::__1::__function::__alloc_func<facebook::react::NativeToJsBridge::runOnExecutorQueue(std::__1::function<void (facebook::react::JSExecutor*)>)::$_8, std::__1::allocator<facebook::react::NativeToJsBridge::runOnExecutorQueue(std::__1::function<void (facebook::react::JSExecutor*)>)::$_8>, void ()>::operator()[abi:ue170006]() [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:193
#61	0x00000001035e604c in std::__1::__function::__func<facebook::react::NativeToJsBridge::runOnExecutorQueue(std::__1::function<void (facebook::react::JSExecutor*)>)::$_8, std::__1::allocator<facebook::react::NativeToJsBridge::runOnExecutorQueue(std::__1::function<void (facebook::react::JSExecutor*)>)::$_8>, void ()>::operator()() at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:364
#62	0x000000010350f134 in std::__1::__function::__value_func<void ()>::operator()[abi:ue170006]() const [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:518
#63	0x000000010350f120 in std::__1::function<void ()>::operator()() const [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:1169
#64	0x000000010350f120 in facebook::react::tryAndReturnError(std::__1::function<void ()> const&) at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/node_modules/react-native/React/CxxModule/RCTCxxUtils.mm:74
#65	0x000000010351a878 in facebook::react::RCTMessageThread::tryFunc(std::__1::function<void ()> const&) at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/node_modules/react-native/React/CxxBridge/RCTMessageThread.mm:69
#66	0x000000010351a644 in std::__1::__function::__value_func<void ()>::operator()[abi:ue170006]() const [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:518
#67	0x000000010351a630 in std::__1::function<void ()>::operator()() const [inlined] at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk/usr/include/c++/v1/__functional/function.h:1169
#68	0x000000010351a630 in invocation function for block in facebook::react::RCTMessageThread::runAsync(std::__1::function<void ()>) at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/node_modules/react-native/React/CxxBridge/RCTMessageThread.mm:45
#69	0x00000001a79aac9c in __CFRUNLOOP_IS_CALLING_OUT_TO_A_BLOCK__ ()
#70	0x00000001a7998dec in __CFRunLoopDoBlocks ()
#71	0x00000001a7998498 in __CFRunLoopRun ()
#72	0x00000001a7997cd8 in CFRunLoopRunSpecific ()
#73	0x0000000103504428 in +[RCTCxxBridge runRunLoop] at /Users/ctoverton/Developer/people/juliannabaker/makeupplaybook/mobile/node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm:337
#74	0x00000001a68cf428 in __NSThread__start__ ()
#75	0x000000020441a06c in _pthread_start ()
  ```
  
</details>

```json
"react-native-gesture-handler": "2.16.2",
"react-native-reanimated": "3.11.0",
```

What's really wierd is I'm fairly sure I have a production build with these same versions and have no issues, but I tried to publish a new build with these and am now having issues.

**Comment by FacuBenito** [comment]
This is what Sentry is telling me about the last error I posted. I don't know if this is relevant, but I hope this helps solve the issue

![Screenshot 2024-09-16 at 9 56 38 AM](https://github.com/user-attachments/assets/4258fe85-f8a9-4bab-9a00-87fca8046e40)

**Comment by FacuBenito** [comment]
Upon further investigation, I found that removing the GestureHandler from the render tree prevented it from crashing. So, I'm guessing that the issue comes from mixing both libraries. I'm currently using `react-native-gesture-handler` in version 2.16.1

**Comment by FacuBenito** [comment]
It turns out a reanimated error is also happening on other parts of the app. Still, I have found no solution to any of the errors I mentioned. I'm kind of running out of options here
![Screenshot 2024-09-17 at 12 11 58 PM](https://github.com/user-attachments/assets/7dd1421f-43f3-4983-9c94-309df4af3795)
![Screenshot 2024-09-17 at 12 13 21 PM](https://github.com/user-attachments/assets/5e6ec541-ece9-4ff4-aab0-fbd4c158cf54)

**Comment by bartlomiejbloniarz** [comment]
Hi! It's seems to be related to [this issue](https://github.com/software-mansion/react-native-reanimated/issues/6114). It was caused by RN missing some methods in their `WithRuntimeDecorator` implementation. We use it to ensure no concurrent access to the runtime. This was fixed in RN 0.75 via [this PR](https://github.com/facebook/react-native/pull/45042) and [this PR](https://github.com/facebook/react-native/pull/45049).
